src/components/Register/style.scss

```diff
@@ -1,245 +1,261 @@
-		padding: 9px;
-		font-size: 17px;
-		outline: none;
-
-		&:focus {
-			border: solid 1px $secondaryblue;
-		}
-	}
-
-	select {
-		-webkit-appearance: none;
-		-moz-appearance: none;
-		appearance: none;
-		border: none;
-		outline: none;
-		background: none;
-		border-radius: 5px;
-		border: solid 1px $gray2;
-		margin-bottom: 5px;
-		padding: 9px;
-		font-size: 17px;
-		outline: none;
-
-		&:focus {
-			border: solid 1px $secondaryblue;
-		}
-	}
-
-	&.confirm-details {
-		input, select {
-			border: solid 1px $light-green;
-		}
-
-		.text {
-			color: $light-green;
-
-			span.info {
-				display: none;
-			}
-		}
-	}
-
-	input:focus ~ p {
-		color: $secondaryblue;
-	}
-
-	.header {
-		margin-top: 13px;
-		margin-bottom: 5px;
-		font-family: Lato;
-		font-size: 20px;
-		font-weight: bold;
-		text-align: center;
-		color: $--secondaryblack;
-	}
-
-	.input-large {
-		width: 383px;
-	}
-
-	.input-major {
-		width: 242px;
-		margin-right: 9px;
-	}
-
-	.input-year {
-		width: 125px;
-	}
-
-	.text {
-		font-size: 11px;
-		font-family: 'Lato', sans-serif;
-		font-weight: bold;
-		color: $--secondaryblack;
-		text-align: left;
-		margin-bottom: 5px;
-
-		span.info {
-			font-size: 0.9em;
-			font-weight: normal;
-			color: $gray2;
-		}
-	}
-
-	.align {
-		display: inline-block;
-	}
-
-	.btn {
-		display: inline-block;
-		margin: 0 auto;
-		margin-top: 5px;
-	}
-
-	@media (max-width: 450px) {
-		.inner { padding: 0px 20px; }
-		.input-large, .input-major {
-			max-width: 383px;
-			width: calc(100% - 17px);
-			margin-right: 0px;
-		}
-		.align.align-input-major { display: block; }
-	}
+  .inner {
+    max-width: 400px;
+    width: 100%;
+    margin: auto;
+    text-align: center;
+    box-sizing: border-box;
+  }
+
+  input {
+    border-radius: 5px;
+    border: solid 1px $gray2;
+    height: 20px;
+    margin-bottom: 5px;
+    padding: 9px;
+    font-size: 17px;
+    outline: none;
+
+    &:focus {
+      border: solid 1px $secondaryblue;
+      background-color: white;
+    }
+  }
+
+  select {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    border: none;
+    outline: none;
+    background: none;
+    border-radius: 5px;
+    border: solid 1px $gray2;
+    margin-bottom: 5px;
+    padding: 9px;
+    font-size: 17px;
+    outline: none;
+
+    &:focus {
+      border: solid 1px $secondaryblue;
+    }
+  }
+
+  &.confirm-details {
+    input,
+    select {
+      border: solid 1px $light-green;
+    }
+
+    .text {
+      color: $light-green;
+
+      span.info {
+        display: none;
+      }
+    }
+  }
+
+  input:focus ~ p {
+    color: $secondaryblue;
+  }
+
+  .header {
+    margin-top: 13px;
+    margin-bottom: 5px;
+    font-family: Lato;
+    font-size: 20px;
+    font-weight: bold;
+    text-align: center;
+    color: $--secondaryblack;
+  }
+
+  .input-large {
+    width: 383px;
+    background-color: white;
+  }
+  .input-invalid {
+    width: 383px;
+    border: 1px solid #ff0000;
```

Comment:
I think we use an existing variable for the red color that’s consistent across the UI. You should use that instead, or create a variable if it doesn’t exist.

Reply:
yup `src/styles/vars.scss` has a lot of styling variables to keep things consistent. Your probably looking for $light-red